libs/barista-components/experimental/datepicker/src/timeinput.ts

```diff
@@ -90,6 +92,46 @@ export class DtTimeInput {
   }
   private _minute: number | null = null;
 
+  /** The minimum selectable hour. */
+  @Input()
+  get minHour(): string | number | null {
+    return this._minHour;
+  }
+  set minHour(value: string | number | null) {
+    this._minHour = value;
+  }
+  private _minHour: string | number | null = null;
```

Comment:
It does not seem that there is a lot of logic going on, is there a reason why this input has getter and setters instead of `@Input() minHour: string | number | null = null`

Reply:
Ah, true. No, I will change that.